fem/fe_coll.cpp

```diff
@@ -1504,7 +1508,7 @@ H1_FECollection::H1_FECollection(const int p, const int dim, const int btype)
    MFEM_VERIFY(p >= 1, "H1_FECollection requires order >= 1.");
    MFEM_VERIFY(dim >= 0 && dim <= 3, "H1_FECollection requires 0 <= dim <= 3.");
 
-   const int pm1 = p - 1, pm2 = pm1 - 1, pm3 = pm2 - 1;
+   const int pm1 = p - 1, pm2 = pm1 - 1, pm3 = pm2 - 1, pm4 = pm3 -1;
```

Comment:
You should declare pm4 later, only where it is actually needed

Reply:
Thanks - I just committed this change and the other about using `new` for `tbeTemp`.  I think the plan is to remove the changes to the examples and the convergence test before this PR is merged.  Let me know how to proceed.